flexx/app/_tornadoserver.py

```diff
@@ -676,15 +676,18 @@ def check_origin(self, origin):
         #WebSocketHandler.check_origin
         
         serving_host = self.request.headers.get("Host")
-        serving_hostname = serving_host.split(':')[0]
-        connecting_host = urlparse(origin).netloc
+        serving_hostname = serving_host.split(':')[0].lower()
+        connecting_host = urlparse(origin).netloc.lower()
+
         
         if serving_hostname == 'localhost':
             return True  # Safe
         elif serving_hostname == '0.0.0.0':
             return True  # we cannot know if the origin matches
```

Comment:
While you're in here, can you please add the test for equal ports for the `0.0.0.0` case as I wrote in the example?
I'll see what to do with the protocol at a later time.